Makefile

```diff
@@ -12,8 +12,8 @@ usage: # Print Targets
 	@grep '^[^#[:space:]].*:' Makefile
 
 check: # Run the tests
-	/bin/bash -c 'py.test -vv -l --tb=auto --full-trace --color=auto tests/test_*.py'
 	/bin/bash -c 'py.test-3 -vv -l --tb=auto --full-trace --color=auto tests/test_*.py'
+#	/bin/bash -c 'py.test -vv -l --tb=auto --full-trace --color=auto tests/test_*.py'
```

Comment:
Why comment it out instead of just removing it?

Reply:
i was not sure about calling the testing tool: `py.test` vs `py.test-3`

Reply:
Is `py.test-3` the name given to the tool by your distro? If I install it via `pip` into a local env, it's just called `pytest`.

I usually advocate project-local installation, because then everybody is on the same page and it is more portable. So if it's possible, use the tool name provided via `pip`.